thebergamo/react-native-fbsdk-next
# feat: Export predefined app event names and parameters

"Standard Events" such as "Complete Registration" and "Standard Event Parameters" such as "Registration Method" are predefined in each native SDKs and only accessible through the SDK: https://developers.facebook.com/docs/app-events/reference

This PR aims to export these values to enable log standard events.

Test Plan:
1. Install packages
2. Check predefined event names and parameters are accessible through `AppEventLogger`
```jsx
import { AppEventsLogger } from "react-native-fbsdk";

AppEventsLogger.logEvent(AppEventsLogger.AppEvents.CompletedRegistration, {
  [AppEventsLogger.AppEventParams.RegistrationMethod]: "email",
});
```
